python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/system.j2

```diff
@@ -20,20 +20,28 @@ system control-plane
 {%     endif %}
 {#     control_plane access_groups ipv4 #}
 {%     for acl_set in system.control_plane.ipv4_access_groups | arista.avd.natural_sort %}
```

Comment:
ipv4_access_groups is list of dicts so we should have the key name base on we need to sort but again there is no required key so we will have issue. Can you please check then do we need to remove sorting here?